test/__mocks__/save.and.continue.mock.ts

```diff
@@ -1,7 +1,5 @@
-import { FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME } from "../../src/config";
 import {
-  CONTINUE_BUTTON_TEXT,
   SAVE_AND_CONTINUE_BUTTON_TEXT
 } from '../__mocks__/text.mock';
 
-export const saveAndContinueButtonText = FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME === "true" ? SAVE_AND_CONTINUE_BUTTON_TEXT : CONTINUE_BUTTON_TEXT;
+export const saveAndContinueButtonText = SAVE_AND_CONTINUE_BUTTON_TEXT;
```

Comment:
Is this constant that references another constant even needed now? Places that use it could just import and use `SAVE_AND_CONTINUE_BUTTON_TEXT`.

Reply:
done